### ISSUE TITLE
apic start fails with error

### ISSUE THREAD

**Issue by metin** [ibm-apiconnect/cli#1]
When executing `apic start`, I am getting this error

```
C:\Users\is96933\work\apis3>apic start
Error: EBUSY: resource busy or locked, rmdir 'C:\Users\is96933\.apiconnect\strong-pm'
```

My setup us like this
- Windows 7 x64, Enterprise
- apic -v => API Connect: v5.0.4.0-iFix2 (apiconnect: v2.3.10)
- npm -v => 3.10.8
- node -v => v4.6.1

**Comment by smartmouse** [comment]
It says resource is locked, so may be there are process still using it.
Try apic stop --all and see if there are existing process running or even try to kill any node.js  process from control panel
What type did projects do you have?
Have you tried the latest APIC V5.0.6.x?

<img width="1256" alt="screen shot 2017-07-17 at 7 35 04 pm" src="https://user-images.githubusercontent.com/4519627/28294363-ab590aca-6b27-11e7-86c2-9cc2b3f6cf05.png">

**Comment by hmoobed** [comment]
Can you turn off the firewall temporarily and try it again and let me know the results.  I think what is happening is that the firewall is blocking some of the ports needed for internal communication.

**Comment by gxvigo** [comment]
Hi, @smartmouse @hmoobed did you find a solution for this?
A customer I am working with, has the same exact error.

Cheers

**Comment by yahelnachum** [comment]
@gxvigo @metin 

Another reason you might be experiencing the `EBUSY` error is if you have left anything open in the `strong-pm` folder, such as the `start.log` or you have an explorer window open to the `strong-pm` folder.

If anything was left open while the user issued an `apic stop` and a subsequent `apic start` command then they would encounter the `EBUSY` error  during the `apic start` command. I was able to recreate this issue on my local Windows 10 machine.

To solve this issue, I closed the `.apiconnect/strong-pm/start.log` and closed the explorer window that was viewing the `strong-pm` folder. I issued the `apic start` command and was able to see the `microgateway` working again.
